locksmith/src/config/config.ts

```diff
@@ -20,6 +20,12 @@ const config = {
         ? 'https://locksmith.unlock-protocol.com'
         : 'https://staging-locksmith.unlock-protocol.com',
   },
+  storage: {
+    endpoint: process.env.STORAGE_ENDPOINT,
+    accessKeyId: process.env.STORAGE_ACCESS_KEY_ID,
+    secretAccessKey: process.env.STORAGE_SECRET_ACCESS_KEY,
+    publicHost: process.env.STORAGE_PUBLIC_HOST,
```

Comment:
Same here?

Reply:
`accessKey` and `secretAcessKey` needs to be protected so we can't make them public. Public host and endpoint will change based on whether you are running in staging or production. We could hard code them along with env and use the correct one based on unlock_env, I guess.